packages/framework/attributor/src/encoders.ts

```diff
@@ -29,10 +28,10 @@ export const deltaEncoder: TimestampEncoder = {
 		}
 		return deltaTimestamps;
 	},
-	decode: (encoded: Jsonable) => {
+	decode: (encoded: unknown) => {
 		assert(
 			Array.isArray(encoded),
-			0x4b0 /* Encoded timestamps should be an array of nummbers */,
+			0x4b0 /* Encoded timestamps should be an array of numbers */,
```

Comment:
Let's fix [here](https://github.com/microsoft/FluidFramework/blob/main/packages/runtime/test-runtime-utils/src/assertionShortCodesMap.ts#L779) too. That file would be regenerated during the assert tagging process when we do the next release to match the fixed comment, but might as well do it now.